pkg/lib/listing.less

```diff
@@ -33,55 +33,55 @@ table.listing-ct > thead td > a {
     padding: @listing-ct-padding 0 @listing-ct-padding;
 }
 
-table.listing-ct thead th {
+table.listing-ct > thead th {
```

Comment:
Have you validated that the rest of Cockpit behaves well with these changes? @garrett Could you give it a look over too?